doc/integrator/fulltext_search.rst

```diff
@@ -32,11 +36,18 @@ Here is an example of an insertion in the ``tsearch`` table:
        to_tsvector('french', 'text to search', ' ', 'g'))
     );
 
-Where ``Layer group`` is the name of the layer group that should be activated,
-``text to display`` is the text that is displayed in the results,
+Where the point geometry contains the coordinates to zoom to,
+``Group`` will be used a group header in the search result display,
+is the name of the layer group that should be activated,
 ``test to search`` is the text that we search for,
 ``french`` is the language used.
 
+The search group header (in the database, ``layer_name``) must always be provided for the
+result to appear in the search results.
+
+After inserting new group headers, you should invalidate the server cache for the new group
+headers to be taken into account.
```

Comment:
Shouldn't it be merged with `Group`?

I'm affrade that it's complicate to understand, for the same field you say:
- `Group`
- search group header (in the database, ``layer_name``) 
- group headers

Or I'm wrong?

Reply:
You are right, I had a mix-up of terms, I update the pull request.
However, in my view it is still a bit confusing because we have a column 'layer_name' in our schema, but it's not the layer.

Reply:
Note if action is empty, is is considered as the layer to add to map : https://github.com/camptocamp/c2cgeoportal/blob/master/geoportal/c2cgeoportal_geoportal/views/fulltextsearch.py#L174:L175